src/pages/bid/bid.html

```diff
@@ -19,25 +19,34 @@
           <ion-col text-center col-2>Score</ion-col>
         </ion-row>
       </ion-grid>
-    </ion-item>
-    <ion-item *ngFor="let state of round.state ; let last = last">
-      <ion-grid>
-        <ion-row>
-          <ion-col col-6>
-            <ion-icon name="logo-buffer" *ngIf="last"></ion-icon> {{state.player}}
-          </ion-col>
-          <ion-col text-center col-2 (click)="setBid(state)">
-            {{state.bid}}
-          </ion-col>
-          <ion-col text-center col-2>
-            {{state.trick}}
-          </ion-col>
-          <ion-col text-center col-2>
-            {{state.score}}
-          </ion-col>
-        </ion-row>
-      </ion-grid>
-    </ion-item>
+    </ion-item-divider>
+    <ion-item-group>
+      <ion-item-sliding *ngFor="let state of round.state ; let last = last; let i = index">
+        <ion-item no-lines [color]="i%2 != 0 ? 'light' : ''">
```

Comment:
Is this in general a good idea to put the logic here to add the classes, cant css solve this for you?